driven/kustomize/base/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: supersecret
+type: Opaque
+data:
+  username: YmxhaA==
```

Comment:
Should we add a note somewhere, possibly a comment in this file, about secrets not being an ideal resource to store in git repositories for security reasons? Or could we simply use a different resource here, for instance a config map?

Reply:
anyway,.... changed to config maps